app/templates/components/add-cron-job.hbs

```diff
@@ -1,32 +1,70 @@
-<form {{action (perform save) on="submit"}}>
-  <div class="form-elem select">
-    <span class="label">Branch</span>
-    {{#x-select value=selectedBranch required="true" on-change=(action (mut selectedBranch)) as |xs|}}
-      {{#each branches as |branch|}}
-        {{#xs.option value=branch}}{{branch.name}}{{/xs.option}}
-      {{/each}}
-    {{/x-select}}
+{{#travis-form 
+  class='form--cron' 
+  onSubmit=(action 'handleSubmit') as |form| }}
```

Comment:
Just for consistency, please put `as` and closing brackets to new lines, for better readability:
```
{{#travis-form
  class='form--cron'
  onSubmit=(action 'handleSubmit') 
  as |form| 
}}
```